src/styles/blocks/card.scss

```diff
@@ -0,0 +1,95 @@
+.card {
+  padding: 0 16px 16px;
+  display: flex;
+  flex-direction: column;
+  color: $main-accent;
+  border-radius: 5px;
+  border: 1px solid $border-color;
+  background: $background-color;
+  transition: transform 300ms ease;
+
+  &__img {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin-top: 32px;
+    margin-right: 3px;
+    margin-left: 3px;
+  }
+
+  &__title {
+    margin-top: 40px;
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    transition: color 300ms ease;
+  }
+
+  &__description {
+    margin-top: 4px;
+    color: $secondary-color;
+    font-size: 10px;
+    font-weight: 400;
+  }
+
+  &__stars-container {
+    display: flex;
+    justify-content: space-between;
```

Comment:
you can move these lines to the separated file like `mixins.scss` and reuse them